src/status_im/contexts/wallet/common/activity_tab/events.cljs

```diff
@@ -68,3 +68,11 @@
      {:db (-> db
               (update-in [:wallet :ui :activity-tab :request] dissoc requestId)
               (update-in [:wallet :activities address] nested-merge activities-indexed))})))
+
+(rf/reg-event-fx
+ :wallet/blockchain-health-changed
+ (fn [{:keys [db]} [{:keys [message]}]]
+   (let [full-status (->> message
+                          (transforms/json->clj)
```

Comment:
I believe we already called `json->clj` on the argument before passing it here

Reply:
i think `message` is a property from the object that was parsed before.

Reply:
Now I see that it was `js->clj` before)